grails-app/controllers/org/pih/warehouse/reporting/ConsumptionController.groovy

```diff
@@ -521,7 +519,7 @@ class ShowConsumptionCommand {
     }
 
     Float getNumberOfMonths() {
-        return numberOfDays / 30
+        return numberOfDays * 30
```

Comment:
The calculation does not correspond to the method name. Please add a separate method if there is a need for this type of calculation.

Reply:
fixed